README.md

```diff
@@ -61,38 +61,35 @@ If you have a capable (Nvidia, at least 8GB of VRAM) GPU, it is highly recommend
+To get the output of this in a slightly better formatted way, you can also look up the option definitions top of **`DeepSpeech.py`**. For executing pre-configured training scenarios, there is a collection of convenience scripts in the `bin` folder. Most of them are named after the corpora they are configured for. Keep in mind that the other speech corpora are *very large*, on the order of tens of gigabytes, and some aren't free. Downloading and preprocessing them can take a very long time, and training on them without a fast **GPU (GTX 10 series recommended)** takes even longer. If you experience **GPU OOM errors** while training, try reducing **`batch_size`.**
 
 As a simple first example you can open a terminal, change to the directory of the DeepSpeech checkout and run:
 
 ```bash
 $ ./bin/run-ldc93s1.sh
 ```
 
-This script will train on a small sample dataset called LDC93S1, which can be overfitted on a GPU in a few minutes for demonstration purposes. From here, you can alter any variables with regards to what dataset is used, how many training iterations are run and the default values of the network parameters.
-Feel also free to pass additional (or overriding) `DeepSpeech.py` parameters to these scripts.
-Then, just run the script to train the modified network.
+This script will train on a small sample dataset called **LDC93S1**, which can be overfitted on a GPU in a few minutes for demonstration purposes. From here, you can alter any variables with regards to what dataset is used, how many training iterations are run and the default values of the network parameters. Feel also free to pass additional (or overriding) **`DeepSpeech.py`** parameters to these scripts. Then, just run the script to train the modified network.
 
 Each dataset has a corresponding importer script in `bin/` that can be used to download (if it's freely available) and preprocess the dataset. See `bin/import_librivox.py` for an example of how to import and preprocess a large dataset for training with Deep Speech.
 
-If you've run the old importers (in `util/importers/`), they could have removed source files that are needed for the new importers to run. In that case, simply remove the extracted folders and let the importer extract and process the dataset from scratch, and things should work.
+If you've ran the old importers (in `util/importers/`), they could have removed source files that are needed for the new importers to run. In that case, simply remove the extracted folders and let the importer extract and process the dataset from scratch, and things should work.
```

Comment:
As your expressing a **conditional** (_If you've ran..._) and a **result** (_they could have removed source..._) you should use the past perfect of _to run_. The past perfect of _to run_ is _have run_ not _have ran_.